packages/core/lib/global-command-options.js

```diff
@@ -4,6 +4,11 @@ const options = {
     description:
       "Specify the network to use. Network name must exist in the configuration."
   },
+  from: {
+    option: "--from <account>",
+    description:
+      "Specify the account from which to make calls or send transactions."
+  },
```

Comment:
Does this belong here? Should I just move this to a quick separate PR?

Reply:
Yeah! @gnidan. I think this can be a separate PR since it is adding a new property for global options, which of course, can later be used by other commands if needed. I will make another PR for adding the `from` property to the global options list.